integration-tests/src/main/java/org/apache/polaris/service/it/test/PolarisRestCatalogIntegrationTest.java

```diff
@@ -109,6 +109,7 @@
  *     <ul>
  *       <li>{@link FeatureConfiguration#ALLOW_EXTERNAL_CATALOG_CREDENTIAL_VENDING}: {@code false}
  *     </ul>
+ *     CODE_COPIED_TO_POLARIS From Apache Iceberg Version: 1.7.1
```

Comment:
Question: do we need it? It seems this class only extend an iceberg classes, which seems fine.

Reply:
There are code copied from Iceberg per offline. SGTM then.

Reply:
Yup, as part of the code is a direct copy of Iceberg tests, it should be documented.